src/style.css

```
@@ -0,0 +1,57 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars--0 {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars--1 {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars--2 {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars--3 {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars--4 {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars--5 {
+  display: flex;
+  flex-direction: row;
+}
```

```suggestion
.stars {
  display: flex;
  flex-direction: row;
}
```
modifiers will be only responsible for amount of active stars
use nth-child for it